nested_conditionals.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Nested conditionals</title>
+  <meta charset="utf-8">
+  <script>
+    var name = prompt("What is your name?");
+    if (name == "flintstones"){
+      alert("Welcome Pedro Flintstones");
+    } else if (name == "bond") {
```

Comment:
if name == "Bond" this will fail.

https://www.w3schools.com/jsref/jsref_tolowercase.asp